docs/style-guide.rst

```diff
@@ -874,6 +874,28 @@ No::
     x = y+z;
     x +=1;
 
+***************
+Order of Layout
+***************
+
+Layout contract elements in the following order:
+
+1. Import statements
```

Comment:
Should have added pragmas as the first one here.

Reply:
OK @axic will handle that asap.

Reply:
Axic already did it.